crm_compassion/models/account_invoice_line.py

```diff
@@ -21,6 +21,10 @@ class AccountInvoiceLine(models.Model):
     currency_id = fields.Many2one(
         'res.currency', 'Currency', related='invoice_id.currency_id',
         store=True)
+    event_id = fields.Many2one(
+        'crm.event.compassion', 'Event',
+        related='account_analytic_id.event_id', store=True, readonly=True
+    )

```

Comment:
Why use store=True in this case?

Reply:
To put it in the database and be able to search from the object directly